src/plugins/data_source/server/client/configure_client.ts

```diff
@@ -11,23 +11,28 @@ import {
   SavedObjectsErrorHelpers,
 } from '../../../../../src/core/server';
 import { DATA_SOURCE_SAVED_OBJECT_TYPE, CREDENTIAL_SAVED_OBJECT_TYPE } from '../../common';
-import { CredentialSavedObjectAttributes } from '../../common/credentials/types';
+import {
+  CredentialSavedObjectAttributes,
+  UsernamePasswordTypedContent,
+} from '../../common/credentials/types';
```

Comment:
`../../common` works as well

Reply:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/32652829/185523261-995ce797-0dd9-4bd1-9f45-72ebb866917e.png">

Reply:
i will refactor on this to expose directly

Reply:
let's create an issue to track this